docs/pages/installation/minikube.md

```diff
@@ -78,6 +78,8 @@ Use Meshery's CLI to streamline your connection to your Minikube cluster. Config
 {% capture code_content %}$ mesheryctl system config minikube{% endcapture %}
 {% include code.html code=code_content %}
 
+Note: The kubeconfig created by minikube is minified and starting Meshery will fail with TLS issues (an invalid certificate). Please run this command to flatten the kubeconfig file: `kubectl config view --minify --flatten > ~/.kube/config-2 && rm ~/.kube/config && mv ~/.kube/config-2 ~/.kube/config`. This command will replace the minified kubeconfig file with the flattened kubeconfig file.
```

Comment:
What about running the command that we built for this? 

`mesheryctl system config`

Reply:
I think that would be even better.

But this is what I am getting:
![Screenshot from 2024-07-15 19-27-33](https://github.com/user-attachments/assets/8f01265b-ef6e-4d49-8d5d-bcccbe2f71c7)

It's looking for auth.json file.

Also `mesheryctl system config` command is not [documented here](https://docs.meshery.io/guides/mesheryctl/system-commands)